pkg/helper/bridges.go

```diff
@@ -0,0 +1,31 @@
+package helper
+
+import (
+	"fmt"
+
+	"github.com/tidwall/gjson"
+
+	yaml "sigs.k8s.io/yaml"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+func getBridgesUp(desiredState nmstatev1alpha1.State) ([]string, error) {
+	foundBridges := []string{}
+
+	desiredStateYaml, err := yaml.YAMLToJSON([]byte(desiredState))
+	if err != nil {
+		return foundBridges, fmt.Errorf("error converting desiredState to JSON: %v", err)
+	}
+
+	queryResults := gjson.ParseBytes(desiredStateYaml).
```

Comment:
Interesting approach :D I'd expect us to convert it to native Go structure and then iterate that. Won't pushback since it is a temporary workaround.

Reply:
I like this approach better for this cases, even for future policies implementation when you have to query the nmstate yaml to get values.

Reply:
Granted we keep it as bytes, you might be right. Would make it way easier to implement for sure.